MobileCenter/MobileCenter/Internals/MSServiceCommon.h

```diff
@@ -38,7 +43,7 @@
  * @discussion: IMPORTANT, This string is used to point to the right storage value for this service.
  * Changing this string results in data lost if previous data is not migrated.
  */
-@property(nonatomic, readonly) NSString *storageKey;
+@property(nonatomic) NSString *storageKey;
```

Comment:
We should think about adding the `copy` attribute to properties of type `NSString` everywhere, including this one. I'm creating a task for this. It's really quick to do.